src/table/components/TableHeadWrapper.jsx

```diff
@@ -40,6 +40,11 @@ function TableHeadWrapper({
     },
   };
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
+  const tableSortLabelStyle = {
+    '&.Mui-active .MuiTableSortLabel-icon': {
+      color: headerStyle.sortLabelColor,
+    },
+  };
```

Comment:
This is fine, but I have a refactor idea where we move all classes made for sx to styling-utils. So you get everything already packaged, and only updated when needed